cypress/integration/mocked/okta_change_password.spec.ts

```diff
@@ -0,0 +1,322 @@
+/**
+ * Test Scenarios:
```

Comment:
I really like this way of summarising the tests, it helped me understand what was happening a lot faster